src/Packagist/WebBundle/Entity/PackageRepository.php

```diff
@@ -279,7 +279,7 @@ public function getFilteredQueryBuilder(array $filters = array())
             $qb->leftJoin('v.tags', 't');
         }
 
-        $qb->orderBy('p.id', 'DESC');
```

Comment:
Well, the ordering is stable here because always applies an internal sorting last, ensuring stable ordering. But this is a non-standard behavior. SQL does not define anything like that (for instance, in Postgresql, the order of elements returned when there is no order by clause is very likely to change between executions, making it unusable in a paginated query).
And I'm not even sure that MySQL guarantees BC about what the internal sorting is. So keeping an explicit sorting would be better (the sorting based on abandoned is not enoguh to make things stable, as there are thousands of ex-aequo for this field)